internal/wrphandlers/mocktr181/handler.go

```diff
@@ -181,20 +184,34 @@ func (h Handler) get(names []string) (int64, []byte, error) {
 	return statusCode, payload, nil
 }
 
-func (h Handler) set(parameters []Parameter) int64 {
+func (h Handler) set(parameters []Parameter) (int64, []byte, error) {
+	result := Tr181Payload{}
 	for _, parameter := range parameters {
-		for _, mockParameter := range h.parameters {
+		for i := range h.parameters {
+			mockParameter := &h.parameters[i]
```

Comment:
I think we meant to use `mockParameter` as a reference, otherwise the value update will not stick (i.e. mock GET will get the default value since it'll never update)